src/os.h

```diff
@@ -277,6 +277,10 @@ class HeapTagScope {
   uword old;
 };
 
+#ifdef TOIT_FREERTOS
+extern bool use_spiram_for_heap;
```

Comment:
Maybe this should be accessed through a static method on OS? Seems more consistent with the rest of the system.